static/app/components/profiling/flamegraph/flamegraphDrawer/flamegraphTreeTable.tsx

```diff
@@ -176,6 +177,42 @@ export function FlamegraphTreeTable({
     [contextMenu, formatDuration, referenceNode, getFrameColor]
   );
 
+  // This is slighlty unfortunate and ugly, but because our two columns are sticky
+  // we need to scroll the container to the left when we scroll to a node. This
+  // should be resolved when we split the virtualization between containers and sync scroll,
+  // but is a larger undertaking and will take a bit longer
+  const onScrollToNode: UseVirtualizedTreeProps<FlamegraphFrame>['onScrollToNode'] =
+    useCallback(
+      (
+        node: VirtualizedTreeRenderedRow<FlamegraphFrame> | undefined,
+        scrollContainer: HTMLElement | null,
+        coordinates?: {depth: number; top: number}
+      ) => {
+        if (node) {
+          const lastCell = node.ref?.lastChild?.firstChild as
+            | HTMLElement
+            | null
+            | undefined;
+          if (lastCell) {
+            lastCell.scrollIntoView({
+              block: 'nearest',
+            });
+
+            const left = -328 + (node.item.depth * 14 + 8);
```

Comment:
This will go away once I virtualize the lists separately, but 328 is width of the sticky columns here...